pkgs/top-level/python-packages.nix

```diff
@@ -6934,11 +6934,11 @@ in modules // {
   };
 
   chardet = buildPythonPackage rec {
-    name = "chardet-2.1.1";
+    name = "chardet-2.3.0";
 
     src = pkgs.fetchurl {
       url = "http://pypi.python.org/packages/source/c/chardet/${name}.tar.gz";
-      md5 = "295367fd210d20f3febda615a88e1ef0";
+      md5 = "25274d664ccb5130adae08047416e1a8";
```

Comment:
I've used `md5` because somewhere it was mentioned that we use hashes from `pypi` when available -- so do I understand correctly that we prefer `sha256` generated by ourselves after all? (I myself do like using `sha256` more)